io/zenoh-transport/src/unicast/mod.rs

```diff
@@ -117,6 +122,11 @@ impl TransportUnicast {
         Ok(transport.get_links())
     }
 
+    pub fn get_auth_ids(&self) -> ZResult<Vec<AuthId>> {
```

Comment:
In some part of the code we use `get_auth_identifiers` while here we use the shorter version. I'd keep the naming 
consistent and I'm more inclined to keep the short version `get_auth_ids`.

Reply:
The long version `get_auth_identifier` is implemented at link level and returns a single `LinkAuthId`.
The short version `get_auth_ids` is implemented a transport level and returns multiple AuthIds (since the transport can control multiple links and have user/password auth enabled).
I believe it's better to have long and short versions instead of having `get_auth_id` and `get_auth_ids` which look too similar to the reader.

Reply:
Renamed all `get_auth_identifier` to `get_auth_id`.